sign.js

```diff
@@ -158,6 +158,9 @@ function signApplicationAsync (opts) {
       } else {
         args.push('--timestamp')
       }
+      if (opts['signature-size'] && Number.isInteger(opts['signature-size'])) {
```

Comment:
Perhaps we can `debugwarn()` some information if a signature size is provided but isn't integer-typed?

Reply:
@sethlu added `debugwarn()` and a minimal check for positive value.